### PR TITLE
Update CQD-upload-tenant-building-data.md

### PR DESCRIPTION
**Pull request by greganth1**
Reflect that there are seven rows and not six in the endpoint file.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/CQD-upload-tenant-building-data.md [hunk 1]
```diff
@@ -156,23 +156,23 @@ The other type of CQD tenant data file is the **Endpoint** data file. The column
...
 
   **Field order:**
 
-  EndpointName, EndpointModel, EndpointType, EndpointLabel1, EndpointLabel2,  EndpointLabel3
+  EndpointName, EndpointMake, EndpointModel, EndpointType, EndpointLabel1, EndpointLabel2,  EndpointLabel3
 
   **Sample row:**
 
-  `1409W3534, Fabrikam Model 123, Laptop, IT designated 2018 Laptop, Asset Tag 5678, Purchase 2018,`  
+  `1409W3534, Fabrikam, Model 123, Laptop, IT designated 2018 Laptop, Asset Tag 5678, Purchase 2018
```
Comment (JohanFreelancer9) [inline Teams/CQD-upload-tenant-building-data.md:175]:
```suggestion
  `1409W3534, Fabrikam, Model 123, Laptop, IT designated 2018 Laptop, Asset Tag 5678, Purchase 2018`  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by SerdarSoysal** [comment]
@microsoftheidi this is CQD, please take a look.

**Pull request merged**